.env

```
@@ -1,6 +1,9 @@
 DB_NAME=campus_db
 DB_ROOT_PASSWORD=secret_root_password
 DB_PORT=3306
+APNS_KEY_ID=
+APNS_TEAM_ID=
+APNS_P8_FILE_PATH=/secrets/AuthKey_XXXX.p8
```

```suggestion
APNS_P8_FILE_PATH=
```

Should this one also be empty and to be filled out by the user?

---

Just wanted to give the user a hint how this should look like.

---

OK 👍🏻